### ISSUE TITLE
Target of target not working

### ISSUE THREAD

**Issue by jcsanady** [Bluefissure/Dalamud-OBS#25]
No blur is being added to the target of target name.  You can easily reproduce this by targeting yourself, or fighting any mob with aggro.

**Comment by Bluefissure** [comment]
It works on my end yesterday, if you click the `Draw Blur Rect` checkbox does it mark the boundary of the target of target?

**Comment by jcsanady** [comment]
No, no blue rectangle with that setting enabled.  No luck toggling the TargetTarget setting, restarting the game, or even reinstalling the addon.  FocusTarget works fine, for whatever that's worth.

**Comment by Bluefissure** [comment]
> No, no blue rectangle with that setting enabled. No luck toggling the TargetTarget setting, restarting the game, or even reinstalling the addon. FocusTarget works fine, for whatever that's worth. I'm on the 3.0.2.2 (64 bit) version of OBS and 1.0.4.0 for the plugin.

Does the target work?

There might be some issue with different UI layout of the game I guess? Are you using a differnt UI layout? Like controller or something different than usual?

If you run `/xldev` and open the dalamud data manager, go to the addon inspector, and then open the focused addon, when you click the target of target addon is there a `TargetTarget` addon shown?

**Comment by Bluefissure** [comment]
Okay I guess there're two different types of the target addon, `_TargetInfo` and `_TargetInfoMainTarget`, probably the target of target doesn't exist in `_TargetInfoMainTarget` addon, I'll look into this later.

**Comment by jcsanady** [comment]
Target blurring also does work, it's just target of target that doesn't.  I'm using the default UI for keyboard and mouse.  Under Addon Inspector in /xldata targetting myself adds _TargetCursor and _TargetInfo to Units 16, no sign of _TargetInfoMainTarget.

I'm going through my list of plugins to see if one of them is conflicting somehow, but it may take some time since I'm busy pushing savage prog this week.  I already went through the only ones I can imagine touching the target bar in any way without luck, though.

**Comment by Bluefissure** [comment]
Can you take a pickture of your node list of `_TargetInfo`?

**Comment by jcsanady** [comment]
Interesting!  It is not.  The last two Text Nodes both show a red box over my target, and I have no idea what it could correspond to.
![image](https://github.com/user-attachments/assets/44fe12e9-4c42-4ea6-a697-1307a0ccc521)
![image](https://github.com/user-attachments/assets/a4f5128c-200a-405d-9a17-97e31921fa45)

**Comment by Bluefissure** [comment]
> Interesting!  It is not.  The last two Text Nodes both show a red box over my target, and I have no idea what it could correspond to.
> 
> ![image](https://github.com/user-attachments/assets/44fe12e9-4c42-4ea6-a697-1307a0ccc521)
> 
> ![image](https://github.com/user-attachments/assets/a4f5128c-200a-405d-9a17-97e31921fa45)
> 
> 

Can you test by targeting yourself?

**Comment by jcsanady** [comment]
Same thing; I was actually targeting myself in the screenshot I provided.  I've noticed that the Distance plugin which adds a range finder to my target actually seems to be stealing the blur, but I missed it before because I was using myself as a target for testing and it doesn't show then.  Just uninstalling it doesn't seem to fix my problem, but with the nightmare gauntlet of world travel right now I haven't wanted to restart the game with it off.  I'll check it out in the morning.  Definitely seems to be just a plugin conflict.
![image](https://github.com/user-attachments/assets/7d64939f-2c12-43ac-8c63-ad0d4fead942)
